shared/js/background/classes/ad-click-attribution-policy.js

```diff
@@ -65,7 +77,11 @@ export class AdClickAttributionPolicy {
         const linkFormat = this.getMatchingLinkFormat(resourceURL)
         if (!linkFormat) return
 
-        const adClick = new AdClick(this.navigationExpiration, this.totalExpiration)
+        const adClick = new AdClick(this.navigationExpiration, this.totalExpiration, this.allowlist)
+
+        if (manifestVersion === 3) {
+            adClick.adClickDNR = new AdClickDNR(tab._tabState.tabId, this.allowlist)
```

Comment:
Don't access _tabState directly
```suggestion
            adClick.adClickDNR = new AdClickDNR(tab.tabId, this.allowlist)
```

Reply:
`tab.id`